pkg/services/cosmosdb/common-types.go

```diff
@@ -58,47 +56,3 @@ type Credentials struct {
 	PrimaryConnectionString string `json:"primaryConnectionString,omitempty"`
 	PrimaryKey              string `json:"primaryKey,omitempty"`
 }
-
```

Comment:
Adding this comment here because github doesn't let me comment unmodified code...

Several lines up from here, we define some types that are just empty structs for binding params and secure binding params. As of about three weeks ago, the broker is tolerant of nil binding params, so you can safely skip defining these types that are unused. Functions that return those types (e.x. `GetEmptyBindingParameters()`) can safely return `nil`.

I do not recall how tolerant the code is of nil `BindingDetails`. You can try. If it doesn't work, we can fix that in a separate PR. But this definitely works for params.